automatic/webstorm/README.md

```diff
@@ -21,3 +21,9 @@ WebStorm - The smartest JavaScript IDE. Lightweight yet powerful IDE, perfectly
 
 - Free 30 day Trial. Beyond that you need to purchase a license.
 
+## Package Parameters
+- `/InstallDir:`- Installation directory, defaults to the 'Program Files (x86)\DataGrip*' directory.
```

Comment:
Wrong stuff here - datagrip.

Reply:
@majkinetor Good catch! fixed in 497847da